code/modules/admin/admin.dm

```diff
@@ -787,9 +787,17 @@ GLOBAL_VAR_INIT(gamma_ship_location, 1) // 0 = station , 1 = space
 	if(GLOB.gamma_ship_location == 1)
 		fromArea = locate(/area/shuttle/gamma/space)
 		toArea = locate(/area/shuttle/gamma/station)
+		for(var/obj/machinery/door/airlock/hatch/gamma/H in GLOB.airlocks)
+			H.locked = FALSE
+			H.update_icon()
```

Comment:
```suggestion
			H.unlock(TRUE)
```
Might want to check if this works instead of snowflaking the doors open

Reply:
tested, it does not.